package.json

```diff
@@ -22,6 +22,8 @@
     "url": "https://github.com/comicrelief/component-library.git"
   },
   "dependencies": {
+    "@contentful/rich-text-react-renderer": "^15.17.0",
```

Comment:
This is just adding a lot of weight to the CL that we don't really need - I'd much prefer (if not _insist_) all of the transforms happen _within_ the CRcom, and the _output_ is passed to the component, just like we do for all the other components here.